tuftool/src/root.rs

```diff
@@ -242,34 +241,11 @@ impl Command {
         exponent: u32,
     ) -> Result<()> {
         let mut root: Signed<Root> = load_file(path)?;
-
-        // ring doesn't support RSA key generation yet
-        // https://github.com/briansmith/ring/issues/219
-        let mut command = std::process::Command::new("openssl");
-        command.args(&["genpkey", "-algorithm", "RSA", "-pkeyopt"]);
-        command.arg(format!("rsa_keygen_bits:{}", bits));
-        command.arg("-pkeyopt");
-        command.arg(format!("rsa_keygen_pubexp:{}", exponent));
-
-        let command_str = format!("{:?}", command);
-        let output = command.output().context(error::CommandExec {
-            command_str: &command_str,
-        })?;
-        ensure!(
-            output.status.success(),
-            error::CommandStatus {
-                command_str: &command_str,
-                status: output.status
-            }
-        );
-        let stdout =
-            String::from_utf8(output.stdout).context(error::CommandUtf8 { command_str })?;
-
-        let key_pair = parse_keypair(stdout.as_bytes()).context(error::KeyPairParse)?;
-        let key_id = hex::encode(add_key(&mut root.signed, roles, key_pair.tuf_key())?);
         key_source
-            .write(&stdout, &key_id)
```

Comment:
I think this may be the only place we do a `key_source.write()` correct?  If so, let's look harder at eliminating this method from `key_source` as mentioned.

Reply:
Yes, this is the only place its used.